Make calendar cloud platform neutral

Bugsbane opened the pull request:
I've tried to take out references to any cloud platform, although there are a couple of places where that didn't quite seem possible, so I've made those references to "ownCloud/Nextcloud". Links to docs etc have been kept the same.

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

The documentation link still point to OC docs, as there's no real platform neutral place to point people to.

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

Commit:
Remove specific reference to ownCloud

Commit:
Make cloud references platform neutral

DeepDiver1975 commented:
@Bugsbane there are a lot of files like .calcontroller.js.kate-swp in this pr - please remove them - THX

Commit:
Remove temporary -swp files

Bugsbane commented:
Sorry about that. ...still learning! So does this look better?

MariusBluem commented on README.md:
```diff
@@ -1,14 +1,14 @@
 [![Dependency Status](https://www.versioneye.com/user/projects/57333497a0ca35004cf77c92/badge.svg?style=flat)](https://www.versioneye.com/user/projects/57333497a0ca35004cf77c92)
 
-**A calendar app for [ownCloud](https://owncloud.org).**  
+**A calendar app for open cloud platforms like [ownCloud](https://owncloud.org) and [Nextcloud](http://nextcloud.com).**  
```
"for ownCloud and Nextcloud" seems to be fine. No need for the "open cloud platforms like" stuff ;)

MariusBluem commented on README.md:
```diff
@@ -1,14 +1,14 @@
+**A calendar app for open cloud platforms like [ownCloud](https://owncloud.org) and [Nextcloud](http://nextcloud.com).**  
 
-This is the new calendar app that has replaced the [old calendar app](https://github.com/owncloudarchive/calendar) with ownCloud 9.0, and is only the frontend for the new CalDAV-Backend which has moved into the [ownCloud Core](https://github.com/owncloud/core). 
+This is the new calendar app that has replaced the [old calendar app](https://github.com/owncloudarchive/calendar), and is only the frontend for the new integrated CalDAV-Backend.
```
Integrated into what? - "for the new server-integrated CalDAV-Backend describes it better I think :wink:

MariusBluem commented on appinfo/info.xml:
```diff
@@ -8,7 +8,7 @@
 		<owncloud min-version="9.0" max-version="9.2" />
 	</dependencies>
-	<description>Calendar GUI for ownCloud's integrated CalDAV server</description>
+	<description>Calendar GUI for the integrated CalDAV server</description>
```
Integrated into what? - "for the new server-integrated CalDAV-Backend describes it better I think :wink:

MariusBluem commented on issue_template.md:
```diff
@@ -25,16 +25,16 @@ Tell us what happens instead
 **PHP version:**
 
-**ownCloud version:** (see ownCloud admin page)
+**ownCloud / Nextcloud version:** (see your admin page)
```
Just call it "server" :wink:

MariusBluem commented on issue_template.md:
```diff
@@ -44,15 +44,15 @@ paste the results here.
 If you have access to your command line run e.g.:
 sudo -u www-data php occ config:list system
-from within your ownCloud installation folder
+from within your cloud installation folder
```
`server` is better then `cloud` I think :grin:

MariusBluem commented on issue_template.md:
````diff
@@ -94,9 +94,9 @@ Eventually replace sensitive data as the name/IP-address of your LDAP server or
 ```
 
-#### ownCloud log (data/owncloud.log)
+#### log file (data/owncloud.log)
````
Log instead of log ... Capital letters at the beginning :wink:

MariusBluem commented:
See my comments @Bugsbane :wink:

Commit:
Add suggestions from @Mar1u5

As per his suggestions in:
https://github.com/owncloud/calendar/pull/699#discussion-diff-74709329
https://github.com/owncloud/calendar/pull/699#discussion-diff-74709350

Commit:
Add suggestions from @Mar1u5

As per:
https://github.com/owncloud/calendar/pull/699#discussion-diff-74709387
https://github.com/owncloud/calendar/pull/699#discussion-diff-74709400
https://github.com/owncloud/calendar/pull/699#discussion-diff-74709417

Bugsbane commented:
Ok, I like all those suggestions, @mar1u5 . Implemented.

MariusBluem commented:
You have missed one :sweat_smile: ... :+1: for the fast changes.

Commit:
Added Mar1u5's last suggestion

At https://github.com/owncloud/calendar/pull/699#discussion-diff-74709362

MariusBluem commented:
THX. Great Initiative :facepunch: ... @georgehrke

georgehrke commented on js/package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "calendar",
-  "description": "Calendar Application for ownCloud",
+  "description": "Integrated Calendar Application",
```
please change to `Calendar Application for ownCloud/Nextcloud`

georgehrke commented:
Looks good except for the one I commented 👍

Bugsbane commented:
> please change to Calendar Application for ownCloud/Nextcloud

The idea is to eliminate using "ownCloud/Nextcloud" wherever possible, to keep the wording as platform neutral as possible. To that end, we've already eliminated a number of the other places this was used with a better alternative. I mean, it's only ownCloud and Nextcloud at the moment, but who's to say what could happen (eg some other platform adds support for OC/NC apps or a project changes their name, or another fork or...) and we're back doing this all again.

Right now, the calendar app description is being displayed in the ownCloud and soon the Nextcloud app store. Saying that an app there is for ownCloud / Nextcloud is redundant. It's similar to having every app in the Google Play store state that it works on Android, or every app on iTunes state that it works on iOS. That's understood and assumed already, and adding it in just distracts from what is actually unique about the app.

If we use the description to describe what the app does, I'm sure people will understand by where it's hosted, what platform it's for.

georgehrke commented:
The `js/package.json` is something only relevant to developers. Users will never ever see this file.

MariusBluem commented:
> I mean, it's only ownCloud and Nextcloud at the moment, but who's to say what could happen (eg some other platform adds support for OC/NC apps or a project changes their name, or another fork or...) and we're back doing this all again.

I think you are better informed then me, if you think that there is any project planning to support apps coded for ownCloud/Nextcloud, or that anybody wants to rename their product ... Another fork?! Nah ... I think this is not going to happen ... There are no reasons currently 😄 @Bugsbane 

The `owncloud/calendar` project, and especially the maintainers (for example @georgehrke or @raghunayyar) are **not** going to make a neutral product which is independent itself, and works on nearly every platform and fork IMHO. This is working for ownCloud and Nextcloud. And the current plan is, to keep it compatible for both projects!

georgehrke commented on issue_template.md:
```diff
@@ -44,15 +44,15 @@ paste the results here.
 If you have access to your command line run e.g.:
 sudo -u www-data php occ config:list system
-from within your ownCloud installation folder
+from within your server installation folder
```
please remove the word server so it's consistent with line 47 and line 74

MariusBluem replied:
I think we should call it here, and in every other line (47, 74, ...) `instance folder`😕 What do you think? @georgehrke

georgehrke replied:
Yes, like that :)
👍

Bugsbane commented:
> I think you are better informed then me, if you think that...

I have no idea what's going to happen in the future, which is why I'm uncomfortable with assuming. Even if nothing changes though "for ownCloud/Nextcloud" is still redundant information for describing an app on the ownCloud/Nextcloud app stores.

> I think we should call it here, and in every other line (47, 74, ...) instance folder

Makes sense. I'll update to add that.

Commit:
"server installation" > "instance installation"

As per https://github.com/owncloud/calendar/pull/699#discussion_r74913034

MariusBluem commented:
:+1: from me ... Lets ask @georgehrke for a final review :wink:

MariusBluem commented:
...and @DeepDiver1975

raghunayyar commented:
:+1: from my end too. Thx for the initiative.

raghunayyar commented:
can you squash the commits please? @georgehrke what do you think?

georgehrke commented:
No need to squash, we can just use GitHub's squash and merge feature

georgehrke commented on issue_template.md:
```diff
@@ -71,7 +71,7 @@ Insert your config.php content here
 With access to your command line run e.g.:
 sudo -u www-data php occ ldap:show-config
-from within your ownCloud installation folder
+from within your installation folder
```
this one needs to be changed to `instance installation folder` as well :)

Commit:
"installation folder">"instance's installation folder"

As per https://github.com/owncloud/calendar/pull/699#discussion-diff-75082204

MariusBluem commented:
Can you change the repository description from 
`📅 Calendar app for ownCloud 9+` to `📅 Calendar app for ownCloud/Nextcloud 9+` ?

@georgehrke

georgehrke commented:
Yes, will do

georgehrke commented:
merged manually
https://github.com/owncloud/calendar/commit/667a861045d5a53bb32302a811a2c89b53b324dc

Bugsbane commented:
Thanks for all your help with this @mar1u5 / @georgehrke
